default.nix

```diff
@@ -287,8 +289,8 @@ let
 
           stylish-haskell = self.callHackageDirect {
             pkg = "stylish-haskell";
-            ver = "0.11.0.3";
-            sha256 = "0rnvcil7i9z7ra2b4znsychlxdj6zm4capdzih1n1v0jp9xi31ac";
+            ver = "0.12.0.0";
```

Comment:
the version used in the latest master branch is [0.12.2.0](https://github.com/haskell/haskell-language-server/blame/master/stack-8.6.5.yaml#L57)